extensions/VisualEditor/modules/ve-mw/dm/nodes/ve.dm.MWTransclusionNode.js

```diff
@@ -69,6 +69,25 @@ ve.dm.MWTransclusionNode.static.getHashObject = function ( dataElement ) {
 	};
 };
 
+/**
+ * return the type to be assigned to given data element. If this is a portable infobox, the proper
+ * node type was already defined and named.
+ *
+ * @param domElements array of clicked elements
+ * @param converter Converter object
+ * @returns {string} type of Node
+ */
+ve.dm.MWTransclusionNode.static.getDataElementType = function ( domElements, converter ) {
+	var isInline = this.isHybridInline( domElements, converter);
+
+	if ( isInline ) {
+		return ve.dm.MWTransclusionInlineNode.static.name;
+	}
+	return this.name === ve.dm.WikiaInfoboxTransclusionBlockNode.static.name ?
```

Comment:
`toDataElement` is called on `ve.dm.MWTransclusionNode` because `ve.dm.Converter.prototype.createDataElements` gets in params the `modelClass` equal to `VeDmMWTransclusionNode()`. It's because `modelName = this.modelRegistry.matchElement( childNode, aboutGroup.length > 1 );` in `ve.dm.Converter.prototype.getDataFromDomSubtree` is equal to `mwTransclusion` for all templates. I think that's why the 'isInline' check was performed here before.